model/Core/Vocabularies/RelationshipType.md

```diff
@@ -72,6 +72,7 @@ name completes the sentence:
 - hasTestCase: Every `to` Element is a test case for the `from` Element (`from` hasTestCase `to`).
 - hasVariant: Every `to` Element is a variant the `from` Element (`from` hasVariant `to`).
 - invokedBy: The `from` Element was invoked by the `to` Agent, during a LifecycleScopeType period (for example, a Build element that describes a build step).
+- instanceOf: The `from` Element (which may be a software `Package` or `Component`) is an instance of the `to` software Component.
```

Comment:
Do we need the added complexity of allowing a component to be an instance of another component?  In terms of tooling, it will require chasing a potentially long chain of sub-components to arrive at the properties that apply to the package.  I also think of `instance` as being something concrete where a `Component` is abstract.

Reply:
Yes, this is needed, because one might have hierarchies of `Component`. I don't think anyone will use a chain more than two levels, but these are needed. More on this on the reply below.